mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -1185,6 +1200,89 @@ def answer_drop_view(self, statement):
 
         return ExecuteAnswer(answer_type=ANSWER_TYPE.OK)
 
+    def answer_create_kb(self, statement: CreateKnowledgeBase):
+        project_name = statement.name.parts[0] if len(statement.name.parts) > 1 else self.session.database
+        # get project id
+        try:
```

Comment:
maybe better to move some code from this method (checks for example) into controller.add ?